Version-02/02-Fundamentals-Part-2/starter/assignmentsFundamentalsP2.js

```diff
@@ -0,0 +1,253 @@
+console.log(populations); // (4) [10, 83, 5, 1441]
+
+// 3. Create an array called 'percentages' containing the percentages of the world population for these 4 population values.
+// Use the function 'percentageOfWorld1' that you created earlier to compute the 4 percentage values
+
+const percentages = ['Portugal', 'Germany', 'Finland', 'China'];
+
+function percentageOfWorld1(population) {
+    return (population / 7900) * 100;
+}
+
+// Call 'percentageOfWorld1' for 3 populations of countries of your choice, store the results into variables, and log them to the console.
+const percPortugal = percentageOfWorld1(10);
+const percGermany = percentageOfWorld1(83);
+const percFinland = percentageOfWorld1(5);
+const percChina = percentageOfWorld1(1441);
+
+console.log(percPortugal, percGermany, percFinland, percChina); // 0.12658227848101267 1.0506329113924051 0.06329113924050633 18.240506329113924
+*/
+
+/* The Solution:
+
+const populations = [10, 1441, 332, 83];
+
+console.log(populations.length === 4);
+const percentages = [
+    percentageOfWorld1(populations[0]),
+    percentageOfWorld1(populations[1]),
+    percentageOfWorld1(populations[2]),
+    percentageOfWorld1(populations[3]),
+];
+console.log(percentages); // Uncaught ReferenceError: percentageOfWorld1 is not defined at assignmentsFundamentalsP2.js:195
+
+// This solution dont work due to the above error.
+*/
+
+/************************************************************************************************************************************* 
+
+42. Intro to Objects:
+
+1. Create an object called 'myCountry' for a country of your choice, containing prooperties 'country', 'capital', 'language', 'population', and
+'neigbours' (an array like we used in previous assignments).
+
+*************************************************************************************************************************************/
+/*
+const myCountry = {
+    country: 'England',
+    capital: 'London',
+    language: 'English',
+    population: 67,
+    neighbours: ['Sally', 'Garry', 'Rachel', 'Jole', 'David', 'Martin', 'Ruth']
+};
+*/
+
+/************************************************************************************************************************************* 
+ 43. Dot vs. Bracket Notation:
+ 
+ 
+1. Using the object from the previous assignment, log a string like this to the console:
+'Finland has 6 million finnish-speaking people, 3 neighbouring countries and a capital called Helsinki.'
+
+2. Increase the country's population by two million using dot notation, and then decrease it by two million using brackets notation.
+
+**************************************************************************************************************************************/
+
+// 1. Answer:
+const jonasCountry = {
+    country: 'Finland',
+    capital: 'Helsinki',
+    language: 'finnish',
+    population: 6,
+    neighbouringCountries: ['Norway', 'Russia', 'Norway']
```

Comment:
Norway is twice in the array, not needed, remove the extra one as it isn't neighbouring country